### DISCUSSION TITLE
Batch reset/recovery options for older versions

### DISCUSSION THREAD

**Discussion by Ating24** [idaholab/moose#26835] [Q&A Tools]
the batch-reset/restore options used one process per app.Can we modify the old version of the batch reset/restore options to enable it to use multiple processes per app?

**Comment by milljm** [comment]
This will not be possible. MOOSE has already created this feature in later versions of MOOSE.

We cannot alter git-history to add an existing feature without a massive negative impact to others.

**Reply by Ating24** [reply]
Can I make modifications on my MOOSE to modify the batch reset/restore options

**Reply by milljm** [reply]
Oh yes, you are welcome to make any change you wish to your clone of the MOOSE repository. If you do decide to go that route, you may be hard pressed finding developers to assist you in doing so however.

I think all developers will answer like minded; to update to the version of MOOSE which adds support for that feature request.

**Reply by grmnptr** [reply]
Here is the pull request that enabled this functionality, you can follow the changes here if you were close enough with your old version. 
https://github.com/idaholab/moose/pull/17643

**Reply by Ating24** [reply]
When the number of processors exceeds the number of examples, the initializing equation system is always displayed
